pages/[id].js

```diff
@@ -1,9 +1,9 @@
 import { useRouter } from "next/router";
 import RecipeDetails from "@/components/RecipeDetails/RecipeDetails";
-import { StyledRecipeHeader, BackLink, StyledRecipeBy, StyledHerbie, StyledHerbieBox, StyledStar, HerbieStarBox, ContentContainer, WhiteSpace
+import { AuthorName, StyledRecipeHeader, BackLink, StyledRecipeBy, StyledHerbie, StyledHerbieBox, StyledStar, HerbieStarBox, ContentContainer, WhiteSpace
```

Comment:
Maybe next time you write the import like this: `import * as Styled from "./xz";` and in the return `<Styled.xy> ` it looks a lot cleaner